### PR TITLE
Changes from working session

### PR DESCRIPTION
**Pull request by landonpierce**

### CODE REVIEW

File: src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Controllers/PermissionsController.cs [hunk 1]
```diff
@@ -24,19 +26,35 @@ public PermissionsController(IPermissionsService permissionsService)
...
     [Route("GetTenantUsers")]
-    public async Task<ICollection<string>> GetTenantUsers(string tenantId)
+    public async Task<IActionResult> GetTenantUsers(string tenantId)
     {
-        return await _permissionsService.GetTenantUsersAsync(tenantId);
+        // Get user IDs from database
+        ICollection<string> userIds = await _permissionsService.GetTenantUsersAsync(tenantId);
+        
+        //
+        // Next, we fetch the user objects with more data from the Microsoft Graph API
```
Comment (asanjabi) [inline src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Controllers/PermissionsController.cs:36]: This is great, we should do more of it

File: src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Interfaces/IGraphAPIService.cs [hunk 2]
```diff
@@ -5,5 +5,6 @@ namespace Saas.Permissions.Service.Interfaces;
 public interface IGraphAPIService
 {
     public Task<string[]> GetAppRolesAsync(ClaimsRequest request);
+    public Task<ICollection<User>> GetUsersByIds(ICollection<Guid> userIds);
```
Comment (asanjabi) [inline src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Interfaces/IGraphAPIService.cs:8]: Why collection and not IEnumerable?

File: src/Saas.SignupAdministration/Saas.SignupAdministration.Web/Program.cs [hunk 3]
```diff
@@ -65,7 +65,10 @@
 
 // builder.Configuration to sign-in users with Azure AD B2C
 builder.Services.AddMicrosoftIdentityWebAppAuthentication(builder.Configuration, Constants.AzureAdB2C)
```
Comment (asanjabi) [inline src/Saas.SignupAdministration/Saas.SignupAdministration.Web/Program.cs:67]: Certainly cleaner than before, my only concern is that all of our other examples use the full URI, might be a little confusing.

File: src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Data/Permission.cs [hunk 4]
```diff
@@ -6,6 +6,7 @@ public class Permission
 {
     public int Id { get; set; }
     public string UserId { get; set; } = string.Empty;
+    public string Email { get; set; } = string.Empty;
```
Comment (asanjabi) [inline src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Data/Permission.cs:9]: Do we still need this?  This one has potential to get stale and out of sync with IDP records.

File: src/Saas.SignupAdministration/Saas.SignupAdministration.Web/Program.cs (hunk 3 above)
Comment (landonpierce) [reply src/Saas.SignupAdministration/Saas.SignupAdministration.Web/Program.cs:67]: I explained this in the project readme already (on the docs-updates branch). I'll clean up our other examples too.

File: src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Interfaces/IGraphAPIService.cs (hunk 2 above)
Comment (landonpierce) [reply src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Interfaces/IGraphAPIService.cs]: Changed 👍

File: src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Data/Permission.cs (hunk 4 above)
Comment (landonpierce) [reply src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Data/Permission.cs]: Ah, thanks for catching that. I meant to remove. Should be gone now

**Pull request closed without merging**